app/src/main/java/cm/aptoide/pt/search/view/SearchResultFragment.java

```diff
@@ -523,8 +524,9 @@ private void setFollowedStoresButtonSelected() {
       allStoresButton.setBackgroundResource(0);
     }
     viewModel.setAllStoresSelected(false);
-    if (defaultThemeName != null && defaultThemeName.length() > 0) {
-      followedStoresButton.setBackgroundResource(StoreTheme.get(defaultThemeName)
+    String storeTheme = viewModel.getStoreTheme();
+    if (storeTheme != null && storeTheme.length() > 0) {
```

Comment:
Could be replaced with something like if (storeThemeExists()) in which storeThemeExists would be a private method. This way readability would be improved